<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SEZ-WS-47" failures="1" tests="4" name="Test" time="11.125" errors="0" timestamp="2023-04-05T09:34:51 IST">
  <testcase classname="testCases.loginTest" name="login" time="2.819">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Login Failed expected [http://main.stage.bt-ms.co] but found [https://ecompartner.digitalmesh.co.in/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Login Failed expected [http://main.stage.bt-ms.co] but found [https://ecompartner.digitalmesh.co.in/]
at pages.loginPage.loginClick(loginPage.java:46)
at testCases.loginTest.login(loginTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testCases.loginTest" name="editProfile" time="0.008"/>
  <testcase classname="testCases.loginTest" name="logoutuser" time="0.246"/>
  <testcase classname="testCases.loginTest" name="userReg" time="0.008"/>
</testsuite> <!-- Test -->
